src/bika/lims/utils/analysisrequest.py

```diff
@@ -267,14 +266,6 @@ def to_list(value):
     # Convert them to a list of service uids
     uids = filter(None, map(to_service_uid, uids))
 
-    # Extend with service uids from profiles
-    profiles = to_list(values.get("Profiles"))
-    if profiles:
-        uid_catalog = api.get_tool(UID_CATALOG)
-        for brain in uid_catalog(UID=profiles):
-            profile = api.get_object(brain)
-            uids.extend(profile.getServiceUIDs() or [])
-
```

Comment:
Great to have this removed. I see that this is already handled by the `setProfiles` method in the sample content.